employdat-app/routes/web.php

```diff
@@ -16,27 +17,29 @@
 | be assigned to the "web" middleware group. Make something great!
 |
 */
+Route::middleware(['auth'])->group(function () {
+    Route::get('/', [PersonController::class, 'index'])->name('home');
+    Route::get('/home', [PersonController::class, 'index']);
+    Route::get('/order/name', [DatsController::class, 'orderName'])->name('order_name');
+    Route::get('/order/email', [DatsController::class, 'orderEmail'])->name('order_email');
+    Route::get('/order/post', [DatsController::class, 'orderPost'])->name('order_post');
+    Route::get('/search', [DatsController::class, 'search'])->name('search_name');
+    Route::get('/insert',function(){return view('insert');})->name('insert');
+    Route::post('/', [PersonController::class, 'store'])->name('add_row');
+    Route::delete('/{id}',[PersonController::class,'destroy'])->name('remove_row');
+    Route::get('/confirm{id}',[DatsController::class,'confirm'])->name('delete_confirm');
+    Route::get('/edit{id}',[PersonController::class,'edit'])->name('edit');
+    Route::patch('/{id}',[PersonController::class,'update'])->name('update');
+
+
+    Route::get('/cv{id}',[DatsCvController::class,'index'])->name('cvs');
```

Comment:
No need to detail the routes, a simple Route::resource('cv', DatsCvController::class) is enough